common/djangoapps/util/tests/test_milestones_helpers.py

```diff
@@ -110,6 +110,11 @@ def test_add_user_milestone_returns_none_when_app_disabled(self):
         response = milestones_helpers.add_user_milestone(self.user, self.milestone)
         self.assertIsNone(response)
 
+    def test_get_service_returns_none_when_app_disabled(self):
```

Comment:
Oh. I think it does on line 14.
`@patch.dict('django.conf.settings.FEATURES', {'MILESTONES_APP': False})`